test/common.js

```diff
@@ -3,7 +3,7 @@
 const path = require('path');
 const fs = require('fs');
 const assert = require('./assert');
-const Sim = require('./../sim');
+const Sim = require('./../dist/sim');
```

Comment:
I thought this kind of thing wasn't going to be necessary?

Reply:
Yeah, I thought it would be easier to do this than to copy over the entire test directory.

Reply:
Feels half-baked to me, but I don't want to hold up fixing the performance issues. A follow-up PR doing something nice with this would be appreciated, I should be able to `rg dist` and see nothing

Reply:
OK.

Reply:
if we are copying over the test directory anyway, we might consider typescripting new test files...

Reply:
We aren't copying over the test directory - that's why there are dist references. We could do that, I suppose, but that seems like something more for the followup.